src/intelligence_layer/core/model.py

```diff
@@ -551,7 +584,9 @@ class AlephAlphaChatModel(ChatModel, ControlModel):
     CHAT_PROMPT_TEMPLATE: PromptTemplate
 
     def to_chat_prompt(
```

Comment:
do you think its **easily** possible to add these controls to this function as well?